documentation/performance/sizing-examples.html

```diff
@@ -166,6 +168,8 @@ <h3>9 nodes, 9 groups with 1 node per group</h3>
       &lt;node hostalias="container1"/&gt;
       &lt;node hostalias="container2"/&gt;
     &lt;/nodes&gt;
+  &lt;/container&gt;
```

Comment:
This one should be "content"

Reply:
Are you sure?
I think the two "container" tags are not closing.
Look:
![image](https://user-images.githubusercontent.com/9601443/72547722-edc52d80-386b-11ea-9741-19b8b0e789c1.png)